src/site/_includes/components/Author.js

```diff
@@ -23,14 +23,14 @@ module.exports = ({id, showSocialMedia = false, small = false}) => {
   if (!author) {
     throw new Error(
       `Can't create Author component for "${id}" without contributor ` +
-        `information. Please check '_data/contributors.js' and make sure the ` +
-        `author you provide is a key in this object.`,
+        "information. Please check '_data/contributors.js' and make sure the " +
+        'author you provide is a key in this object.',
     );
```

Comment:
this seems odd? it's using both double and single quotes

Reply:
I believe this is happening because the string has single quotes that are not escaped. I can update the strings to use single quotes and escape the inner single quotes

Reply:
ah I see.

Can we tell the linter to ignore this entire block and leave it with the backtick strings? We are doing string interpolation on the first line, so it seems like overkill to have to escape.